### ISSUE TITLE
How to use plugin directly with <script>

### ISSUE THREAD

**Issue by falkon** [euvl/vue-notification#31]
Added script in page:
```
<script src="https://unpkg.com/vue"></script>
<script src="https://unpkg.com/vue-notification@1.3.3"></script>
```

Error:
```
vue-notification@1.3.3:145 Uncaught TypeError: __WEBPACK_IMPORTED_MODULE_0_vue___default.a is not a constructor
    at Object.module.exports (vue-notification@1.3.3:145)
    at __webpack_require__ (vue-notification@1.3.3:30)
    at Object.defineProperty.value (vue-notification@1.3.3:196)
    at __webpack_require__ (vue-notification@1.3.3:30)
    at Object.defineProperty.value (vue-notification@1.3.3:163)
    at __webpack_require__ (vue-notification@1.3.3:30)
    at Object.directions.x (vue-notification@1.3.3:515)
    at __webpack_require__ (vue-notification@1.3.3:30)
    at module.exports.rawScriptExports (vue-notification@1.3.3:76)
    at vue-notification@1.3.3:79
```

**Comment by euvl** [comment]
Hi @falkon,

I don't think it should. it is an amd module, you will probably need require.js or smth if you want to use it without webpack.

**Comment by mooyoul** [comment]
## TL;DR
Use below snippet:

```
<script type="text/javascript">
  window.vue = window.Vue; // quick fix for vue-notification
</script>
<script type="text/javascript" src="https://unpkg.com/vue-notification"></script>
<script type="text/javascript">
  window.vueNotification = window["vue-notification"].default;
  Vue.use(vueNotification);
</script>
```


### Explaination
By default, vue should be exported to `global.Vue`, not `global.vue` when CommonJS/AMD is unavailable (in browser, vue should exported to `window.vue`.)

Many 3rd party browsers import vue module using CommonJS or ES import syntax.
when importing module, the package name 'vue' is used for referencing vue package.
This causes that webpack will try to find vue from `global.vue` when any module system is not available.

and, this module using import/export syntax for module. so we need to reference default export value using `.default` property
